index.js

```diff
@@ -1,44 +1,40 @@
 module.exports = {
-    env: {
-        browser: true,
-        node: true,
-        es2024: true,
-        mocha: true
+    name: "tao:js",
+    files: ["**/*.{cjs,mjs,js,svelte}"],
+    languageOptions: {
+        globals: {
+            browser: true,
+            node: true,
+            es2024: true,
+            mocha: true,
+            Cypress: true,
+            cy: true,
+        },
     },
-    globals: {
-        Cypress: true,
-        cy: true
-    },
-    parser: '@babel/eslint-parser',
```

Comment:
removing the parser would be a breaking change for many apps that use babel for linting